### PR TITLE
[#IOPAE-540] Setup B4F and Forward io-services-cms request

### PR DESCRIPTION
**Pull request by giamma1295**
Initial B4F API Bootstrap
#### List of Changes
- Add  placeholders for API B4F described in `backoffice/openapi.yaml`
- Implement `io-services-cms` request proxy
- init unit tests for `io-services-cms` request proxy

#### How Has This Been Tested?
Unit test and application local run

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  API_SERVICES_CMS_MOCKING,
+  API_SERVICES_CMS_URL
+} from "@/config/constants";
+import { Client, createClient } from "@/generated/services-cms/client";
+import { readableReport } from "@pagopa/ts-commons/lib/reporters";
+import * as E from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import { NextRequest, NextResponse } from "next/server";
+
+if (API_SERVICES_CMS_MOCKING) {
```
Comment (gunzip) [inline apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts:12]: why do you want to setup mocks here in the module? can't we do this in the tests files? or in some setting at top level. generally, modules shouldn't have side effects when imported.

File: apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  });
+
+type PathParameters = {
+  serviceId?: string;
+  keyType?: string;
+  limit?: string;
+  offset?: string;
+};
+
+export const forwardIoServicesCmsRequest = async <T extends keyof Client>(
```
Comment (gunzip) [inline apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts:31]: may you add a comment on the purpose of this method?

File: apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts (hunk 1 above)
Comment (giamma1295) [reply apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts:12]: This is done when you want to run the application locally and do not want to call external services APIs, all HTTP request to those external services will be intercepted by MSW and a mock response returned

File: apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts (hunk 2 above)
Comment (giamma1295) [reply apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts]: done

File: apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts (hunk 1 above)
Comment (giamma1295) [reply apps/backoffice/src/app/api/utils/io-services-cms-proxy.ts]: in any case removed

**Review by rodoviti — APPROVED** [review]
LGTM

**Pull request merged**